surface/include/pcl/surface/mls.h

```diff
@@ -52,6 +52,28 @@
 
 namespace pcl
 {
+  /** \brief Data structure used to store the results of the MLS fitting
+    * \note Used only in the case of VOXEL_GRID_DILATION or DISTINCT_CLOUD upsampling
+    */
+  struct MLSResult
+  {
+    MLSResult () : mean (), plane_normal (), u_axis (), v_axis (), c_vec (), num_neighbors (), curvature (), valid (false) {}
```

Comment:
Can we remove the eigen default initializers which do nothing and explicitly set the  value initialized the native types to 0/false?